Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM --platform=linux/amd64 clux/muslrust:stable as builder
+ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
+RUN mkdir /app
```

Comment:
```suggestion
```

`WORKDIR` instruction automatically creates directory if it doesn't exist

Reply:
thanks, removed!